### PR TITLE
Correct pagination

### PR DESCRIPTION
**Pull request by CaroFG**

### CODE REVIEW

**Comment by curquiza** [comment]
Can you explain which bug this PR fixes?

**Comment by CaroFG** [comment]
In the front-end: 
Kaminari pagination didn't work properly because and an error was raised when trying to change the page because the page param was sent as a string instead of an integer. 
Will_paginate would send all the results to all the pages.

File: lib/meilisearch/pagination/will_paginate.rb [hunk 1]
```diff
@@ -8,7 +8,11 @@ module MeiliSearch
   module Pagination
     class WillPaginate
       def self.create(results, total_hits, options = {})
-        ::WillPaginate::Collection.create(options[:page], options[:per_page], total_hits) { |pager| pager.replace results }
+        ::WillPaginate::Collection.create(options[:page], options[:per_page], total_hits) do  |pager|
+          start = (options[:page] - 1 ) * options[:per_page]
```
Comment (curquiza) [inline lib/meilisearch/pagination/will_paginate.rb:12]:
```suggestion
          start = (options[:page] - 1) * options[:per_page]
```

**Review by curquiza — CHANGES_REQUESTED** [review]
small change 😇

**Review by curquiza — APPROVED** [review]
🚀
merge when you want ;)

**Comment by CaroFG** [comment]
bors try

**Comment by CaroFG** [comment]
bors merge

**Pull request merged**